scripts/hooks/prevent-illegal-characters.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Just a best practice. I started changing it in other scripts.